### PR TITLE
Fixed running the application locally (vagrant)

### PR DESCRIPTION
**Pull request by jankowiakmaria**

### CODE REVIEW

**Comment by matteofigus** [comment]
@jankowiakmaria quick question - is it necessary to lock node to that version?

File: Vagrantfile [hunk 1]
```diff
@@ -11,15 +11,15 @@ Vagrant.configure("2") do |config|
...
   config.vm.provision :shell, inline: 'apt-get purge -y chef*'
   config.vm.provision :shell, inline: 'sed -i \'s/^mesg n$/tty -s \&\& mesg n/g\' /root/.profile'
- 
+
   config.vm.network :forwarded_port, guest: 3006, host: 3006
 
   config.vm.provision "docker" do |d|
-    d.pull_images "quay.io/coreos/etcd"
-    d.run "quay.io/coreos/etcd", args: "-p 4001:4001", demonize: true
+    d.pull_images "quay.io/coreos/etcd:v0.4.6"
```
Comment (chriscartlidge) [inline Vagrantfile]: We are currently running different version in prod, i'm not sure if you want to align it with that?

Comment (jankowiakmaria) [inline Vagrantfile]: that's the oldest image available (it used to use default one which was latest)

**Comment by jankowiakmaria** [comment]
@matteofigus it used to use the latest version of node which generated bunch of errors (during creating image - installing dependencies). We've picked the oldest available image from https://hub.docker.com/_/node/

**Comment by matteofigus** [comment]
@jankowiakmaria that may be because the latest is 5, which is unstable. I would rather go for latest 4 - which is stable (while we wait for 6 which should be out in a couple of months)

**Comment by lobut** [comment]
@matteofigus Well, we also decided that because the Manual in the ReadMe said that it's dependent on Node 0.10.26.  I'll do a check to see if it runs on 4 and we can change accordingly.

**Comment by matteofigus** [comment]
Ok, got it. I think it is good for now @lobut @jankowiakmaria - Perhaps we can use node 4 + new etcd in hobknob v3

**Comment by chriscartlidge** [comment]
@matteofigus I agree

**Pull request merged**